### PR TITLE
[skip-ci][DOCS] Freshen up README/INSTALL

### PR DESCRIPTION
**Pull request by eguiraud**
I think it's useful/user-friendly to have basic installation instructions in-source.
I removed the outdated content and added pointers to root.cern/install/build_from_source and similar.

This fixes ROOT-10964 .

### CODE REVIEW

File: README/INSTALL [hunk 1]
```diff
@@ -7,81 +7,49 @@ Contents:
...
 
 2. Getting ready to build:
 ==========================
 
-Building ROOT requires to have CMake in your system. This can be
-downloaded from http://cmake.org
-You may want to compile features into ROOT, which depends on third
-party libraries. Make sure you meet all demands for additional
+ROOT requires CMake to be built.
+You may want to compile features into ROOT, which depend on third-party libraries.
```
Comment (Axel-Naumann) [inline README/INSTALL:35]:
```suggestion
You may want to build ROOT with features (such as HistFactory) that depend on third-party libraries (such as libxml2).
```

File: README/INSTALL [hunk 2]
```diff
@@ -7,81 +7,49 @@ Contents:
...
 
 There are two main methods of installing ROOT from source.
   - 'location independent' is advised for a personal installation of ROOT
-  - 'fix location' for a system wide installation. 
-  
-Both require to set the CMAKE_INSTALL_PREFIX variable at configuration time. 
-The 'fix location' requires in addition to set the 'gnuinstall' option at configuration. 
+  - 'fixed location' for a system wide installation.
+
+Both require to set the CMAKE_INSTALL_PREFIX variable at configuration time.
```
Comment (Axel-Naumann) [inline README/INSTALL:51]: Is that really true? I +/- never use it as I +/- never `make install`.

File: README/INSTALL [hunk 3]
```diff
@@ -150,201 +118,18 @@ Using the fixed location mode (section 3.2), the default 'CMAKE_INSTALL_PREFIX'
 path is '/usr/local', which will result in the ROOT files to be
 installed in '/usr/local/bin', `/usr/local/lib', etc. You can specify
 an installation prefix other than `/usr/local' by giving `cmake'
```
Comment (Axel-Naumann) [inline README/INSTALL:120]: Shall we suggest `/opt/root-6.22/` "as that makes it easy to uninstall ROOT and to keep multiple versions"?

File: README/INSTALL [hunk 4]
```diff
@@ -150,201 +118,18 @@ Using the fixed location mode (section 3.2), the default 'CMAKE_INSTALL_PREFIX'
...
-To build the event generator interface for Venus, you need to have the
-Venus libraries.  The sources can be downloaded from
-
-    ftp://root.cern.ch/root/venus/
-
-
-7.10. Secure Remote Password (SRP) Authentication:
---------------------------------------------------
-Not supported any longer
+the option `-DCMAKE_INSTALL_PREFIX=PATH'.
```
Comment (Axel-Naumann) [inline README/INSTALL:121]:
```suggestion
the option `-DCMAKE_INSTALL_PREFIX=/opt/root-6.22'.
```

File: README/INSTALL (hunk 2 above)
Comment (eguiraud) [reply README/INSTALL:51]: I guess they don't _require_ it, they both _honor_ it (and the default is `/usr/local`). I'll clarify.

File: README/INSTALL (hunk 3 above)
Comment (eguiraud) [reply README/INSTALL:120]: yes!

**Comment by eguiraud** [comment]
will merge after addressing Axel's comments

**Pull request merged**